Character/Classes/Warlock/Warlock.xml

```diff
@@ -344,7 +344,7 @@
 				<text>You can communicate telepathically with your familiar and perceive through your familiar's senses as long as you are on the same plane of existence. Additionally, while perceiving through your familiar's senses, you can also speak through your familiar in your own voice, even if your familiar is normally incapable of speech.</text>
 			</feature>
 		</autolevel>
-		<autolevel level="5">
+		<autolevel level="5" IL="IL">
```

Comment:
These are the newly available ones, so should not have the IL attribute

Reply:
But they are also represented in PS list, why should they stay then?
I mean, as if I understand correctly, excluding IL and leaving PS expects final compendium to not have any invocations inside the warlock class, but in special spell list. But without these tags, invocations stay inside the class, which is different from what is expected.

Reply:
How else can a user automatically remove those lists from his compendium, if not by this method?

Reply:
I apologize, I got otherwise occupied and thought that I had responded to this.

The IL tag is not meant to be used to completely remove the list, its purpose is to remove redundant items when other options become available.

There is not a mechanism in place to completely remove these items, or meta magic options, because they are part of the class and should be displayed for leveling up. The PS list is there for convenience of reference, rather than as a leveling up option.

Reply:
Ok, that have been a false impression. Will fix this. I still require functionality to remove those lists completely though, but that's a thing for another PR.

Reply:
Also, might as well clarify the option's description in the script, because for me "Inlined list" seems more like any of those lists, not the "repeated" part.

Reply:
If you need the functionality for your use, you're obviously free to add that into your fork, you could use the IL attributes on everything if that is what you want your version to be. That's kind of the beauty of being able to pull one's own copy of the repo and customize it to one's own use case, as I am mostly doing with this one. I'm not sure if it is a generally applicable change to have the ability to remove all of these items, but if you think that it is, please write up an issue for it and I will consider its addition and implementation.

I'm sorry that you have found that option confusing, I'm not sure where you looking at the description of the "IL" attribute that doesn't noted "repeated" though.
In the templates file it mentions it as `Inlined-Lists(eg repeated Eldritch Invocations)`, and in the script itself: `IL InlinedLists (eg repeated Eldritch Invocations)`. If there is somewhere that I am missing that provides a confusing description, I will happily clarify it.

Reply:
Thing is, original repo isn't supported for past 3 years, and this one seemed like most updated version.

The reasoning for me why removing all of them is better is because Warlocks can switch invocations each level, which isn't supported at all. And spell UI is better for them, as player can add all invocations they might want and then "prepare" them on levelups; most of them are also more spell-like than most other class features, so its handy to have most abilities in one place (spell page). So, if its better, why would I ever look into features list for them, where they just take a lot of space in the list. Also, a little optimization on xml file size.

I did see "repeated" notion, but I'm not truly native speaker, so I confused myself, it happens with me once in a while. I've read "repeated" like it was about all invocations, cause they repeat themselves... partly because I wanted that to mean what I seeked for, not what it really means.

Reply:
I have been trying to keep it the most up to date, primarily because I want to be able to use shiny, new stuff, but i have been a bit busy and inattentive/not getting notifications

I will concede that it is a better UI for managing them, since you can change them frequently and I think that this is a fine change.